Examples/DicomImagePrintTags/Documentation.rst

```diff
@@ -6,16 +6,37 @@ Read Image Meta-Data Dictionary and Print
 Overview
 --------
 
-This example illustrates how to read only an image's information and meta-data dictionary without loading the pixel content via the `ImageFileReader <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ImageFileReader.html>`_.
+This example illustrates how to read only an image's information and meta-data
+dictionary without loading the pixel content via the
+`ImageFileReader <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ImageFileReader.html>`_.
 
```

Comment:
This looks like some auto formatting happened here. This should not be in the same commit at the other changes.